lua/dap.lua

```diff
@@ -75,6 +78,7 @@ M.adapters = {}
 --
 M.configurations = {}
 
+function M.session() return session end
```

Comment:
When we want to have no change to `dap.lua` UIs need somehow access to the session.